semantic error: Restriction mismatch with provided arguments
  --> tests/errors/generics_error_03.py:19:1 - 22:1
   |
19 |    
   |    ...
...
   |
22 |    
   | ...^ 
